eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  rules: {
+    "space-infix-ops": ["error", {"int32Hint": false}],
```

Comment:
@jrans can we add a comment with a link to the issue where the rule is discussed on each line of the config file? e.g: 
```js
"space-infix-ops": ["error", {"int32Hint": false}], // https://github.com/dwyl/goodparts/issues/32
"max-len": ["error", 80], // https://github.com/dwyl/goodparts/issues/186
```